src/gui/src/settings/options-window.cpp

```diff
@@ -342,6 +343,151 @@ OptionsWindow::OptionsWindow(Profile *profile, ThemeLoader *themeLoader, QWidget
 	ui->comboImagePositionVideoH->setCurrentIndex(positionsH.indexOf(settings->value("imagePositionVideoH", "left").toString()));
 	ui->lineImageBackgroundColor->setText(settings->value("imageBackgroundColor", QString()).toString());
 
+	ui->checkRememberDrawer->setChecked(settings->value("Zoom/rememberDrawer", true).toBool());
+	ui->checkRememberGeometry->setChecked(settings->value("Zoom/rememberGeometry", true).toBool());
```

Comment:
I don't understand the point of the "rememberGeometry" one.
Is there any time where an user would _not_ want to remember the window's geometry?
The drawer makes a little bit more sense since you might want to have it always closed by default.

Reply:
rememberGeometry isn't for a specific case, it's for users who might often resize their windows depending on what they're looking at or what other windows they have open. With this, they don't have to bother changing it back to something reasonable for other pictures later, they can just close the window and it will be back to the way they expect it next time. I've used settings like this in other image viewing programs, like Irfanview, and it is the natural behaviour for many light-weight programs, like sxiv, which just open a simple window.